src/pages/DirectMessage/components/DMChattingList/DMChattingList.tsx

```diff
@@ -1,40 +1,44 @@
 import { useParams } from "react-router-dom"
 import * as S from "./DMChattingList.Styles"
-import { AUTH_API } from "@/apis/Api"
-import { useCallback, useEffect, useState } from "react"
-import { Conversation } from "@/types"
+
+import { Message } from "@/types"
+import useChattingList from "../../hooks/useChattingList"
+import { useEffect, useRef } from "react"
+import DMInput from "./DMInput/DMInput"
+import DMMessageItem from "./DMMessage/DMMessageItem"
+
 const DMChattingList = () => {
-  const { id } = useParams()
-  const [selectedMessageList, setSelectedMessgaeList] = useState([])
+  const { id: othersId } = useParams()
```

Comment:
```suggestion
  const { id: othersUserId } = useParams()
```
[P2] 까지 적히면 더좋을것 같네요